bin/plot_krun_results

```diff
@@ -202,6 +229,9 @@ def main(is_interactive, data_dcts, plot_titles, window_size, outfile,
         fig, export_size = draw_page(is_interactive,
```

Comment:
Yeah, it's been on my list to tray and box these args up into some data structure(s?). But it's probably very low priority.